### ISSUE TITLE
Write introduction tutorials

### ISSUE THREAD

**Issue by VictorLamoine** [codelv/declaracad#29]
Writing tutorials would help standard CAD users to try declarative parametric CAD.

I'm using CAD quite a lot, Onshape, SolidWorks and FreeCAD. I also develop a lot in C++ and despite this I would like to have a reference document on how to get started with `declaracad`.

I can find one OpenCascade tutorial and that's it:
https://www.opencascade.com/doc/occt-7.0.0/overview/html/occt__tutorial.html

**Comment by frmdstryr** [comment]
> I can find one OpenCascade tutorial and that's it

Yes... Unfortunately OpenCascade's business model seems to be selling support / training so the amount of freely available tutorials are limited (which is completely understandable). The only other examples using OCC directly that I know of are https://github.com/tpaviot/pythonocc-demos

> Writing tutorials would help standard CAD users to try declarative parametric CAD.

Agree, could you share some points of confusion that would be good to cover? Or even suggest an outline of tutorials?

**Comment by VictorLamoine** [comment]
I also found:
https://occtutorials.wordpress.com/

Something like `CadQuery` does would be great, of course it takes time to write this kind of documentation:
https://pythonhosted.org/cadquery/index.html

**Comment by frmdstryr** [comment]
I setup a website here and started adding a few things https://declaracad.com/

It takes a lot of time to do proper documentation....
